Sources/ContentType.swift

```diff
@@ -178,4 +178,28 @@ extension ContentType: ResourceQueryable {
                             }
         })
     }
+
+    public func fetch<ResourceType>() async throws -> (Result<ResourceType, Error>, ResponseType)
```

Comment:
The function should be type of ResouceType intead of (Result<ResouceType, Error>)
@vkalta

Reply:
And take same reference for other functions as well @vkalta